README.md

```diff
@@ -1,8 +1,14 @@
-# ert-runner.el [![Build Status](https://api.travis-ci.org/rejeep/ert-runner.el.png?branch=master)](http://travis-ci.org/rejeep/ert-runner.el)
+# ert-runner.el
 
-Ert-runner is a tool for Emacs projects tested using Ert. It assumes a
-certain test structure setup and can therefore make running tests
-easier.
+[![Build Status](https://travis-ci.org/rejeep/ert-runner.el.svg)](https://travis-ci.org/rejeep/ert-runner.el)
+[![Coverage Status](https://coveralls.io/repos/rejeep/ert-runner.el/badge.svg)](https://coveralls.io/r/rejeep/ert-runner.el)
+[![MELPA](http://melpa.org/packages/ert-runner-badge.svg)](http://melpa.org/#/ert-runner)
+[![MELPA Stable](http://stable.melpa.org/packages/ert-runner-badge.svg)](http://stable.melpa.org/#/ert-runner)
+[![Tag Version](https://img.shields.io/github/tag/rejeep/ert-runner.el.svg)](https://github.com/rejeep/ert-runner.el/tags)
```

Comment:
I don't know whether we need that badge.  It'll always be the same as MELPA Stable so it seems somewhat redundant to me.  

Also, I'd prefer a different order of badges.  The text reads from left to right, so the most important badges should come first, i.e. left-most.  Most users won't care for our coverage and build status, but only about the version and probably the license.

Hence the MELPA badges should come first, the license badge next, and the status badges at the very end.